todo_app/data/Trello.py

```diff
@@ -0,0 +1,119 @@
+from todo_app.data.ResponseFilter import ResponseFilter
+from todo_app.ListUtils import swap_key_in_list, combine_lists
+import requests
+import os
+
+
+class trello:
+    api_key = os.environ.get('API_KEY')
+    api_token = os.environ.get('API_TOKEN')
+    trello_board_id = os.environ.get('BOARD_ID')
+
+    todo_name = 'To Do'
+    in_progress = 'Doing'
+    done = 'Done'
+    todo_id = None
+    in_progress_id = None
+    done_id = None
+
+    base_url = 'https://api.trello.com/1/'
+
+    headers = {'key': api_key, 'token': api_token}
+
+    fields_from_cards = {'fields': f'{ResponseFilter.name},{ResponseFilter.id_list}'}
+
+    get_cards_header = headers | fields_from_cards
```

Comment:
This is Python 3.9+ only.
You should specify that requirement in pyproject.toml